src/app/components/Picture/index.tsx

```diff
@@ -0,0 +1,49 @@
+import classNames from 'classnames';
+import React, { useMemo } from 'react';
+import styles from './index.module.scss';
+import { Breakpoint, BreakpointWidths } from 'types/tailwind';
+
+type Sources = {
+  src: string;
+  media: Breakpoint;
```

Comment:
media is not required.
```suggestion
  media?: Breakpoint;
```

Reply:
fixed